Harvest.Web/ClientApp/src/sass/_layout.scss

```diff
@@ -18,7 +18,6 @@ body {
   border-right: 1px solid $borders;
   border-bottom: 1px solid $borders;
   box-shadow: $box-shadow;
-  overflow: hidden;
```

Comment:
Does this mean we no longer have to manually put `overflow: "visible"` on card-wrapper divs that have DateTime and/or Typeahead components?

Reply:
yea